infra-as-code/bicep/modules/vwanConnectivity/vwanConnectivity.parameters.example.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "parVirtualHubEnabled": {
+        "value": true
+      },
+      "parVPNGatewayEnabled":{
+          "value": true
+      },
+      "parERGatewayEnabled":{
+        "value": true
+      },      
+      "parAzureFirewallEnabled": {
+          "value": true
+      },
+      "parCompanyPrefix":{
+          "value": "alz"
+      },
+      "parPublicIPSku":{
+          "value":"Standard"
+      },
+      "parTags":{
+          "value":{
+            "Environment": "POC"
```

Comment:
Do we want this set in case someone leaves it by mistake?

Reply:
this was copied and pasted from hubNetworking module :) will need to fix it there too